docs: update client docs for v0.9.2 website credentials login (#30)
Notifiarr/mkdocs-wiki: docs/pages/client/install.md

1. Open C:\ProgramData and create a folder `notifiarr`
1. Create a new folder named `logs`, so you now have `C:\ProgramData\notifiarr\logs`
    - When you add the log paths in the client UI (later steps), make sure you point them to a file such as:
      - `C:\ProgramData\notifiarr\logs\app.log`
      - `C:\ProgramData\notifiarr\logs\debug.log`

>       - `C:\ProgramData\notifiarr\logs\http.log`

Reviewer: These indents are supposed to be here.

### New Install

1. Download `notifiarr.amd64.exe.zip` from [the Releases page](https://github.com/Notifiarr/notifiarr/releases)
1. Save it in `C:\ProgramData\notifiarr`
1. Open the folder that was created from extracting and copy the `.exe` + example `.conf` files up one directory so it is located at:
- `C:\ProgramData\notifiarr\notifiarr.amd64.exe`
- `C:\ProgramData\notifiarr\notifiarr.conf.example`
1. You can now delete the `.zip` file that was downloaded and the folder that was extracted
1. Rename `notifiarr.conf.example` to `notifiarr.conf`

> 1. Double-click `notifiarr.amd64.exe` to launch the client. The webui will be available at `http://127.0.0.1:5454`

Reviewer: ```suggestion
1. Double-click `notifiarr.amd64.exe` to launch the client. The webui will be available at [http://127.0.0.1:5454](http://127.0.0.1:5454)
```

    - `C:\ProgramData\notifiarr\notifiarr.conf.example`
1. You can now delete the `.zip` file that was downloaded and the folder that was extracted
1. Rename `notifiarr.conf.example` to `notifiarr.conf`
1. Double-click `notifiarr.amd64.exe` to launch the client. The webui will be available at [http://127.0.0.1:5454](http://127.0.0.1:5454)

> 1. Enter your API key when prompted

Reviewer: ```suggestion
1. Enter your API key when prompted
1. Login for the first time with your Notifiarr.com email address and password.
```

1. Copy the existing conf file to `C:\ProgramData\notifiarr\notifiarr.conf`
1. If the `C:\users\<your home folder>\.notifiarr` folder exists, delete it

### First Run & Autostart

- At this point, the structure should look like the [Desired Outcome mentioned above](#desired-outcome).

#### First Run

1. Double-click `notifiarr.amd64.exe` to launch
1. Enter your API key when prompted

> 1. Access the Web UI at `http://127.0.0.1:5454`

Author: ```suggestion
1. Access the Web UI at [http://127.0.0.1:5454](http://127.0.0.1:5454)
```

> 1. Access the Web UI at `http://127.0.0.1:5454`

Reviewer: ```suggestion
1. Access the Web UI at [http://127.0.0.1:5454](http://127.0.0.1:5454)
```

Reviewer: wtf github

- Username: `admin`

> - Password: Your API key

Reviewer: ```suggestion
    - Username: email address you login to notifiarr.com with
    - Password: password for notifiarr.com
1. You may set a local password after you login.
```

#### Autostart

> 1. Right-click on the `.exe` and create a shortcut

Reviewer: ```suggestion
1. Right-click on `notifiarr.amd64.exe` and create a shortcut.
```

1. Press Windows key + R, type `shell:startup`, then select OK
1. Copy the shortcut to the opened Startup folder

#### Finding Password in Logs (if needed)

> - **Option A:** Check `notifiarr.log` (or `app.log`) - password is at the top of the file

Reviewer: this is gone.

> - **Option B:** Right-click the notifiarr tray icon → Logs → View

Reviewer: ```suggestion
- **Option B:** Right-click the notifiarr tray icon → Config → Password - and set a new password.
```

1. Copy the existing conf file to `C:\ProgramData\notifiarr\notifiarr.conf`
1. If the `C:\users\<your home folder>\.notifiarr` folder exists, delete it

> ### First Run & Autostart

Reviewer: Just remove this header since the two below are redundant. Then make the next two `###` instead of `####`.

- At this point, the structure should look like the [Desired Outcome mentioned above](#desired-outcome).

#### First Run

1. Double-click `notifiarr.amd64.exe` to launch
1. Enter your API key when prompted
1. Access the Web UI at [http://127.0.0.1:5454](http://127.0.0.1:5454)
    - Username: email address you login to notifiarr.com with
    - Password: password for notifiarr.com
1. You may set a local password after you login.

#### Autostart

1. Right-click on `notifiarr.amd64.exe` and create a shortcut.
1. Press Windows key + R, type `shell:startup`, then select OK
1. Copy the shortcut to the opened Startup folder

#### Setting a Local Password

> - **Option A:** In the Web UI, go to Config → Password

Reviewer: ```suggestion
- **Option A:** In the Web UI, click your username → Trust Profile
```
